tests/unit_tests/sparse_blas/source/sparse_trsv_usm.cpp

```diff
@@ -0,0 +1,224 @@
+/*******************************************************************************
+* Copyright 2023 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions
+* and limitations under the License.
+*
+*
+* SPDX-License-Identifier: Apache-2.0
+*******************************************************************************/
+
+#include <algorithm>
+#include <complex>
+#include <cstdlib>
+#include <iostream>
+#include <limits>
+#include <vector>
```

Comment:
Are we sure we need all of these?

Reply:
Indeed, I'll remove some of them on Monday.

Reply:
Done in https://github.com/oneapi-src/oneMKL/pull/407/commits/9f420d9e8bb6df5e64bcc7c8b4823247ea33474a